tests/base.py

```diff
@@ -27,8 +27,10 @@ def find_unused_port():
 
 
 class BaseTest(unittest.TestCase):
+
```

Comment:
I got error on make test and make cov so I run https://pypi.python.org/pypi/isort and https://pypi.python.org/pypi/autopep8/